lib/ansible/modules/windows/win_domain.py

```diff
@@ -106,4 +127,18 @@
     safe_mode_password: password123!
     sysvol_path: C:\Windows\SYSVOL
   register: domain_install
+
+- name: Create new Windows domain in an existing forest with specific parameters
+  win_domain:
+    create_dns_delegation: yes
+    database_path: C:\Windows\NTDS
+    dns_domain_name: child
+    domain_mode: Win2012R2
+    domain_netbios_name: ANSIBLE-CHILD
```

Comment:
I think it would be better to not have an example of NetBIOS name different than the DNS name. I know the above has the same issue but regarding new additions, I don't think we want to encourage this usage.